### ISSUE TITLE
Plugin is generating warnings for NaN values

### ISSUE THREAD

**Issue by michaelyork** [lukasroegner/homebridge-dyson-pure-cool#208]
Plugin works great! But logs are littered with the following:

```
[9/2/2021, 10:06:58 PM] [homebridge-dyson-pure-cool] This plugin generated a warning from the characteristic 'PM2.5 Density': characteristic value expected valid finite number and received "NaN" (number). See https://git.io/JtMGR for more info.
[9/2/2021, 10:06:58 PM] [homebridge-dyson-pure-cool] This plugin generated a warning from the characteristic 'PM10 Density': characteristic value expected valid finite number and received "NaN" (number). See https://git.io/JtMGR for more info.
```

Running latest version (2.0.3).

**Comment by Fish2** [comment]
Im also having the same issues Dyson have done a update recently might be related

**Comment by lukasroegner** [comment]
Fixed/Enhanced in version 2.3.4.
